app/__init__.py

```diff
@@ -11,6 +12,10 @@
 
 API_VERSION = "1.0"
 
+# Connect to Agolia
+client = SearchClient.create('your_project_name', 'your_key')
+index = client.init_index('your index name')
+
```

Comment:
I'm not certain how the environment variables are added and we dont have an OC account yet anyways so I left it like this for now.

Reply:
Likely through Travis on prod and through the `config.py`. Do other environment variables currently exist in the application? If so, find their example usages, and I can add the variable for you in Travis.

I forgot what you're supposed to do locally, but you could inline the key into `config.py` as long as you don't commit it.

http://flask.pocoo.org/docs/1.0/config/

Reply:
@kylemh I believe I added the pattern properly. It wont work till travis has the info right? I can give you my info for now if we dont have the OSS account yet

Reply:
I can implement it for you! DM me the key on Slack